### ISSUE TITLE
Tests errors when compiling on Linux aarch64

### ISSUE THREAD

**Issue by adomenech73** [facebook/react#7816]
**Do you want to request a *feature* or report a *bug*?**

bug, but probably on still unsupported arch 

**What is the current behavior?**

I'm trying to compile react on linux aarch64, and get this two tests errors:

```
FAIL  src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js (16.692s)
● ReactComponentTreeHook › stack addenda › it gets created
  - Expected '
        in Constructor (at **)' to be '
        in Unknown (at **)'.
        at jasmine.buildExpectationResult (node_modules/jest/node_modules/jest-cli/node_modules/jest-jasmine2/src/index.js:80:44)
        at Object.eval (src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js:1755:114)
        at tryOnTimeout (timers.js:232:11)
        at Timer.listOnTimeout (timers.js:202:5)

 FAIL  src/isomorphic/classic/element/__tests__/ReactElementValidator-test.js (4.5s)
● ReactElementValidator › it warns for keys for arrays with no owner or parent info
  - Expected 'Warning: Each child in an array or iterator should have a unique "key" prop. Check the top-level render call using <Constructor>. See https://fb.me/react-warning-keys for more information.
        in div (at **)' to be 'Warning: Each child in an array or iterator should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.
        in div (at **)'.
        at jasmine.buildExpectationResult (node_modules/jest/node_modules/jest-cli/node_modules/jest-jasmine2/src/index.js:80:44)
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: https://jsfiddle.net/reactjs/69z2wepo/).**

```
wget https://github.com/facebook/react/archive/v15.3.2.tar.gz
tar xvf v15.3.2.tar.gz
cd react-15.3.2/
npm install
npm install -g grunt
grunt clean
grunt test
```
**What is the expected behavior?**

Pass all tests

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**

Linux aarch64 (arm64) and react version 15.3.2

**Comment by keyanzhang** [comment]
This should have been fixed in master (#7670) but we haven't cut a release with this commit yet. Could you try building the master branch instead?

**Comment by adomenech73** [comment]
@keyanzhang It worked perfectly as you said! I just had to checkout '00ac179' and all the test where passed.

Thanks
